src/main/java/io/cdap/delta/bigquery/Schemas.java

```diff
@@ -113,7 +122,7 @@ private static Field convertToBigQueryField(Schema.Field field) {
     Schema.LogicalType logicalType = fieldSchema.getLogicalType();
     Field.Mode fieldMode = isNullable ? Field.Mode.NULLABLE : Field.Mode.REQUIRED;
     if (logicalType != null) {
-      StandardSQLTypeName bqType = convertLogicalType(logicalType);
+      StandardSQLTypeName bqType = convertLogicalType(fieldSchema);
```

Comment:
why do we need to change the method signature from accepting a logicalType to accepting a Schema ? if the method is meant for converting logical type ?

Reply:
We need additional details while converting logical type such as `precision` and `scale`. These details are not present in the `Schema.LogicalType` but present in the fieldschema.